mache-chroniclemap/src/test/java/com/excelian/mache/chroniclemap/ChronicleMapMacheProvisionerShould.java

```diff
@@ -0,0 +1,88 @@
+package com.excelian.mache.chroniclemap;
+
+import com.excelian.mache.chroniclemap.ChronicleMapMacheProvisioner.ChronicleMapBuilderConfig;
+import com.excelian.mache.core.HashMapCacheLoader;
+import net.openhft.chronicle.map.ChronicleMapBuilder;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.mockito.Mockito;
+
+import java.io.File;
+import java.util.List;
+
+import static com.excelian.mache.chroniclemap.ChronicleMapMacheProvisioner.chronicleMap;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.verify;
+import static org.mockito.internal.util.reflection.Whitebox.getInternalState;
+
+public class ChronicleMapMacheProvisionerShould {
+
```

Comment:
Im also fine with whatever the chosen standard is and don't mind using any given style. Also agree that the instance variables and many methods can be hard to navigate, especially when the private methods have similar names and slightly differing behaviour. 

An interesting if old SO question on the topic has some good answers is:
http://stackoverflow.com/questions/155436/unit-test-naming-best-practices
I particularly like the C# one half way down that starts "I like to follow the.." and his subsequent update a few years later. 
There is also a good comment towards the end relating AAA style testing with naming, something I hand't considered but would agree with (really its a better description of what I described as my personal preference above).